src/components/Header/index.js

```diff
@@ -12,6 +7,8 @@ import HomeIcon from '@material-ui/icons/Home';
 import LocalActivityIcon from '@material-ui/icons/LocalActivity';
 import EventIcon from '@material-ui/icons/Event';
 import PeopleIcon from '@material-ui/icons/People';
+import WorkIcon from '@material-ui/icons/Work';
+/* Uncomment when re-enabling timesheets link */
```

Comment:
Yeah, that's a mistake on my end. Since I do end up using the WorkIcon to create the Timesheets tab, Travis didn't show any errors for unused imports. I've removed the commented code on line 11. The code inside of the render method that displays the Timesheets Tab is still commented out.